apps/juxtaposition-ui/src/services/juxt-web/views/portal/messageThread.tsx

```diff
@@ -0,0 +1,64 @@
+import moment from 'moment';
+import cx from 'classnames';
+import { PortalPageBody, PortalRoot } from '@/services/juxt-web/views/portal/root';
+import { PortalNavBar } from '@/services/juxt-web/views/portal/navbar';
+import { PortalNewPostView } from '@/services/juxt-web/views/portal/newPostView';
+import type { MessageThreadItemProps, MessageThreadViewProps } from '@/services/juxt-web/views/web/messageThread';
+import type { ReactNode } from 'react';
+
+function MessageThreadItem(props: MessageThreadItemProps): ReactNode {
+	const msg = props.message;
+	let content = <p className="post-content">{ msg.body }</p>;
+	if (msg.screenshot) {
+		content = <img className="message-viewer-bubble-sent-screenshot" src={`${props.ctx.cdnUrl}${msg.screenshot}`} />;
+	} else if (msg.painting) {
+		content = <img className="message-viewer-bubble-sent-memo" src={`${props.ctx.cdnUrl}/paintings/${msg.pid}/${msg.id}.png`} />;
+	}
+
+	return (
+		<div
+			id={`message-${msg.id}`}
+			className={cx('post scroll', {
+				'my-post': msg.pid === props.ctx.pid,
+				'other-post': msg.pid !== props.ctx.pid
+			})}
+		>
+			<a href={`/users/show?pid=${msg.pid}`} data-pjax="#body" className="scroll-focus mii-icon-container">
```

Comment:
Not essential for this PR, but we should think about a helper class using URLSearchParams so we don't muck up the encoding in more complex cases (it does URL escaping and stuff)

Reply:
Done!